content/millennium/dstu2/infrastructure/document-reference.md

```diff
@@ -92,8 +92,43 @@ Notes:
 #### Response
 
 <%= headers status: 200 %>
-
```

Comment:
nit: any reason you removed this?

Reply:
All the other response calls in this file have been squished together, just being consistent.

Reply:
fair enough, +1